Pcap++/header/PcapDevice.h

```diff
@@ -18,6 +18,79 @@ namespace pcpp
 	//Forward Declaration - required for IPcapDevice::matchPacketWithFilter
 	class GeneralFilter;
 
+	namespace internal
+	{
+		/// @class PcapHandle
+		/// @brief A wrapper class for pcap_t* which is the libpcap packet capture descriptor.
+		/// This class is used to manage the lifecycle of the pcap_t* object
```

Comment:
nit: can we change all doxygen comments to be `/* ... */` to be consistent with the rest of the code?

Reply:
Sure, tho I am of the opinion that the `/// ...` look better with the less line bloat of extra lines above and below each description.

Btw, #1357 also has `/// ...` type comments.

Reply:
`///` is also a doxygen style. I think we can use both. In addition, I also agree with @Dimi1010  that `///` is simple when there are only a few lines.

Reply:
I prefer consistency in the code base as much as possible, especially because this file contains another class `IPcapDevice` with the other documentation style `/** ... */`

Reply:
Fixed e5bfda6